examples/coolest-todos/composable-apps/profile/src/index.ts

```diff
@@ -1,6 +1,5 @@
-import { createApp } from "@leanjs/react-router";
+import { createApp } from "@leanjs/react/18";
```

Comment:
you need to `import { createApp } from "@leanjs/react-router/18";` if you want the app to have a router.